### DISCUSSION TITLE
💭 Docs and their discussions should be separated for readability

### DISCUSSION THREAD

**Discussion by jonahsnider** [rome/tools#1217] [General]
## The problem

Looking at issues like #3 we have a short issue that tells us exactly what we need to know about diagnostics in Rome, great.

But other issues like #20 are not as concise.  It starts off like #3 but has a long discussion about linting in Rome. Discussions like this are great, but it can make extracting information about Rome's current state harder.
Do I need to read 38+ replies to understand what linting is like? The top-level comment hasn't been edited, is this still accurate?

## My proposed solution

I'm not a current fan of this system and I think a better solution is to separate issues like that into 2 parts:

1. A discussion issue (maybe a GitHub discussion, if you can get them to enable it on this repository)
2. A wiki page that acts as a source of truth for that topic

Having 1 page someone can look at to learn about a topic means you can guarantee what your reading is up to date and something planned for Rome.

I'm not sure how people feel about what I've suggested, and I'm not sure how well this scales to giant projects with large communities like Rome. I would love to get feedback on what you think about my suggestion, defense of the current system, or maybe your own idea.

**Comment by mmkal** [comment]
Looks like discussions were enabled: https://github.com/rome/tools/discussions

**Comment by jonahsnider** [comment]
With discussions enabled following a structure similar to what Next.js has been doing for their big features is probably a good setup (see [image RFC](https://github.com/vercel/next.js/discussions/16832)).

**Comment by sebmck** [comment]
I agree! I did enable GitHub Discussions when I saw they were available. Separating tasks/bugs and discussions will go a really long way to making things more manageable. I'll update our issue templates and convert all the discussion labeled issues to discussions.
